src/main/resources/db/changelog/changes/v2.1.1/harvested-views.xml

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet id="create_view_publication_request" author="your_author_name_here">
```

Comment:
Super small thing, but
```suggestion
    <changeSet id="create_view_publication_request" author="nils.buxel@stud.htwk-leipzig.de">
```

Reply:
Yes, I wasn't sure which address to use yet.. my student mail won't be accessible after graduation.. I will add one!